### PR TITLE
Added card data structure, playfield.

### PR DESCRIPTION
**Pull request by admoore0**

### CODE REVIEW

**Review by kdreese — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # Godot 4+ specific ignores
 .godot/
+
+# Art GIMP files
+_assets
```
Comment (thearst3rd) [inline .gitignore:5]: goofy ahh line not in the local gitignore and muddying the git commit history smh my head

**Review by kdreese — APPROVED** [review]

File: src/cards/attack/attack_card_data.gd [hunk 2]
```diff
@@ -0,0 +1,12 @@
+## A data structure for attack cards.
+class_name AttackCardData
+extends Resource
+
```
Comment (thearst3rd) [inline src/cards/attack/attack_card_data.gd:4]: #pedantry add a second line break

File: src/cards/card.gd [hunk 3]
```diff
@@ -0,0 +1,11 @@
+extends Control
+
```
Comment (thearst3rd) [inline src/cards/card.gd:2]: #pedantry add a second line break

**Review by thearst3rd — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (thearst3rd) [reply .gitignore:5]: you don't need to change this btw

**Review by thearst3rd — APPROVED** [review]

**Pull request merged**